code/game/objects/items/misc_items.dm

```diff
@@ -65,8 +65,8 @@
 	w_class = W_CLASS_TINY
 
 /obj/item/folded_bag/attack_self(mob/user)
-	to_chat(user, "<span class = 'notice'>You unfold \the [src] into a plastic bag.</span>")
-	var/bag = new/obj/item/weapon/storage/bag/plasticbag
+	to_chat(user, "<span class = 'notice'>You unfold \the [src].</span>")
+	var/bag = new/obj/item/weapon/storage/bag/plasticbag(get_turf(user))
```

Comment:
user.loc isn't always a turf, like if they do it in a closet or disposal pipe

Reply:
so, user.loc is what you're wanting it to be instead?

Reply:
yes please